### ISSUE TITLE
Fix element locator issue of test : tests.foreman.ui.test_syncplan.Syncplan.test_positive_update_[1,2]

### ISSUE THREAD

**Issue by jyejare** [SatelliteQE/robottelo#2033]
Unable to find out the correct locator for Edit Button of Sync plan name update.

StackTrace:
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jyejare/Desktop/RedHat/RobotteloTop/robottelo/tests/foreman/ui/test_syncplan.py", line 229, in test_positive_update_1
    self.syncplan.update(plan_name, new_name=new_plan_name)
  File "/home/jyejare/Desktop/RedHat/RobotteloTop/robottelo/robottelo/ui/syncplan.py", line 70, in update
    self.wait_until_element(locators["sp.name_edit"]).click()
AttributeError: 'NoneType' object has no attribute 'click'
-------------------- >> begin captured logging << --------------------
py.warnings: WARNING: /usr/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:79: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.
  InsecurePlatformWarning

py.warnings: WARNING: /usr/lib/python2.7/site-packages/requests/packages/urllib3/connectionpool.py:769: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.org/en/latest/security.html
  InsecureRequestWarning)

root: DEBUG: NoSuchElementException: Could not locate element //div[contains(@class, 'jnotify-notification-error')].
root: DEBUG: TimeoutException: Timed out waiting for element '//form[@bst-edit-text='syncPlan.name']//i[contains(@class,'icon-edit')]' to display.
--------------------- >> end captured logging << ---------------------
```

**Comment by kbidarkar** [comment]
This may  require s/icon-edit/fa-edit/ for the syncplan-name in locators.py

**Comment by jyejare** [comment]
Another test affected due to same issue, 
Test: tests.foreman.ui.test_syncplan.Syncplan.test_positive_update_2
Locator: locators["sp.sync_interval_edit"]

**Comment by jyejare** [comment]
This issue has observed on both RHEL 6.6 and RHEL 7.1

**Comment by jyejare** [comment]
The merged PR #2047 fixed this issue. Hence closing the issue.
